src/commands/project/init.ts

```diff
@@ -80,6 +107,7 @@ export const initCommandDecorator = (command: commander.Command): void => {
  */
 export const initialize = async (
   rootProjectPath: string,
+  variableGroupName: string,
```

Comment:
`variableGroupName` should be optional I think since its not really really required to init a project.